### ISSUE TITLE
'no-extra-parens' fires on jsx code

### ISSUE THREAD

**Issue by ddaza** [yannickcr/eslint-plugin-react#204]
I don't know if this it relevant to report here, but here is the situation. 

As per  eslint bug [#1814](https://github.com/eslint/eslint/issues/1814) the [no-extra-pars] (http://eslint.org/docs/rules/no-extra-parens) rule fires correctly but the react plugin should catch the JSX and basically ignore the rule (right?).  
If I omit the parenthesis I am getting the warning : 
```
|185 col 16 warning| Missing parentheses around 
multilines JSX (react/wrap-multilines) [javascript/eslint]
```
If I do have the parenthesis then I get this error. 

```
|185 col 16 error| Gratuitous parentheses around expression. 
(no-extra-parens) [javascript/eslint]
```

So the obvious work around would be to disable the rule completely, but I believe this inconsistency should be handled by the plugin.

This just started happening when I last updated 
````
eslint@1.3.1
babel-eslint@4.1.1
eslint-plugin-react@3.3.1
```

**Comment by yannickcr** [comment]
Duplicate of #86, sadly these two rules cannot be enabled at the same time.
